src/test/java/org/broadinstitute/hellbender/utils/mcmc/AdaptiveMetropolisSamplerUnitTest.java

```diff
@@ -0,0 +1,106 @@
+                @Override
+                protected double proposalLogLikelihood(Double proposal, DoubleState state, EmptyData data) {
+                    return (a - 1) * Math.log(proposal) + (b - 1) * Math.log(1 - proposal);
+                }
+
+                @Override
+                protected double currentValue(final DoubleState state) {
+                    return state.value();
+                }
+            };
+        ParameterizedModel<DoubleState, EmptyData> betaModel =
+                new ParameterizedModel.GibbsBuilder<>(new DoubleState(BETA_START), EMPTY_DATA, DoubleState.class)
+                .addParameterSampler(DoubleState.PARAMETER_NAME, betaSampler, Double.class).build();
+
+        return new GibbsSampler<>(NUM_BURN_IN_STEPS + NUM_SAMPLES, betaModel);
+        }
+
+
+    // factory for sampling Gaussian with mean mu and precision lambda
+    final static GibbsSampler<DoubleState, EmptyData> gaussianGibbsSampler(final double mu, final double lambda) {
+        final  Sampler<Double, DoubleState, EmptyData> gaussianSampler =
+                new AdaptiveMetropolisSampler<DoubleState, EmptyData>(INITIAL_STEP_SIZE) {
+                    @Override
+                    protected double proposalLogLikelihood(Double proposal, DoubleState state, EmptyData data) {
+                        final double x = state.value();
+                        return -(lambda/2)*(proposal-mu)*(proposal-mu);
+                    }
+
+                    @Override
+                    protected double currentValue(final DoubleState state) {
+                        return state.value();
+                    }
+                };
+
+        final ParameterizedModel<DoubleState, EmptyData> gaussianModel =
+                new ParameterizedModel.GibbsBuilder<>(new DoubleState(BETA_START), EMPTY_DATA, DoubleState.class)
+                .addParameterSampler(DoubleState.PARAMETER_NAME, gaussianSampler, Double.class).build();
+
+        return new GibbsSampler<>(NUM_BURN_IN_STEPS + NUM_SAMPLES, gaussianModel);
+    }
+
+
+    @Test
+    public void testBeta() throws Exception {
+        for (final double a : Arrays.asList(10, 20, 30)) {
+            for (final double b : Arrays.asList(10, 20, 30)) {
+                final double theoreticalMean = a / (a + b);
+                final double theoreticalVariance = a*b/((a+b)*(a+b)*(a+b+1));
+                final double standardDeviationOfMean = Math.sqrt(theoreticalVariance / NUM_SAMPLES);
+
+                final GibbsSampler<DoubleState, EmptyData> sampler = betaGibbsSampler(a, b);
+                sampler.runMCMC();
+                List<Double> samples = sampler.getSamples(DoubleState.PARAMETER_NAME, Double.class, NUM_BURN_IN_STEPS);
+
+                final double sampleMean = samples.stream().mapToDouble(x -> x).average().getAsDouble();
+                Assert.assertEquals(sampleMean, theoreticalMean, 10*standardDeviationOfMean);
```

Comment:
10 standard deviations seems too lax...but this test seems to fail more often than it should when i reduce this.  not sure what's going on here?

should also check standard deviation of the samples.